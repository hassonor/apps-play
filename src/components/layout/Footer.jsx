import {ReactComponent as IronSourceLogo} from '../../assets/IronSource-logo.svg';

function Footer() {
    const currentYear = new Date().getFullYear()

    return (
        <footer className='footer p-3 bg-gray-700 text-primary-content footer-center'>
            <div>
                <IronSourceLogo/>
                <p> &copy; {currentYear} IronSource, Inc.</p>
            </div>
        </footer>
    )
}


export default Footer;