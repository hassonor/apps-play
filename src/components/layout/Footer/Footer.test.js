import React from "react";
import {shallow} from "enzyme";
import Footer from "./Footer";

it("expect to render Footer Component", () => {
    expect(shallow(<Footer/>)).toMatchSnapshot();
});