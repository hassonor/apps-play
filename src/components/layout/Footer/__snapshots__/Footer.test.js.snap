// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Footer Component 1`] = `
<footer
  className="footer p-3 bg-gray-700 text-primary-content footer-center"
>
  <div>
    <ForwardRef(SvgIronSourceLogo) />
    <p>
       Â© 
      2022
       IronSource, Inc.
    </p>
  </div>
</footer>
`;
